src/controllers/router-store/types.ts

```diff
@@ -34,6 +35,7 @@ type PrivateStateProperties = {
   onPrefetch?: (routerContext: RouterContext) => void;
   routes: Routes;
   unlisten: UnlistenHistory | null;
+  plugins: Required<Plugin>;
```

Comment:
Let's call this `combinedPlugins` or something similar so it is more clear that is is the "unified" API and not an array.